docs/eventing/broker-trigger.md

```diff
@@ -116,6 +116,12 @@ namespace.
 kubectl -n default get broker default
 ```
 
+*NOTE* `Broker`s created due to annotation will not be removed if you
+remove the annotation. For example, if you annotate the namespace,
+which will then create the `Broker` as described above. If you now
+remove the annotation, the `Broker` will not be removed, you have
+to manually delete it.
```

Comment:
Add the command to delete it:

For example, to delete the injected Broker from the `foo` namespace:

```shell
kubectl -n foo delete broker default
```

Reply:
done, PTAL.